modules/server/src/network/resolvers/aggregations.ts

```diff
@@ -93,18 +96,28 @@ const convertFieldsToString = (requestedFields: RequestedFieldsMap) => {
 	return gqlFieldsString;
 };
 
+/**
+ * Query string creation
+ *
+ * @param documentName
+ * @param requestedFields
+ * @returns
+ */
 export const createNodeQueryString = (
 	documentName: string,
 	requestedFields: RequestedFieldsMap,
 ) => {
 	const fields = convertFieldsToString(requestedFields);
-	const aggregationsString = !isEmpty(fields) ? `aggregations ${fields}` : '';
-	const gqlString = `{${documentName} { hits { total }  ${aggregationsString} }}`;
+	const aggregationsString = !isEmpty(fields)
+		? `aggregations(filters: $filters, aggregations_filter_themselves: $aggregations_filter_themselves, include_missing: $include_missing) ${fields}`
+		: '';
+	const gqlString = `query nodeQuery($filters: JSON, $aggregations_filter_themselves: Boolean, $include_missing: Boolean) {${documentName} { hits { total }  ${aggregationsString} }}`;
```

Comment:
we are declaring 3 variables here... if they are not provided in the args object will GQL throw an error?

Reply:
it's ok for these. default gql args are nullable. there would be an error if args were required using '!' eg. `(filter: JSON!)`